src/components/exercises/Splash.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { View, Text, Dimensions } from 'react-native';
+import { RoundImageWithBorder} from '../Components';
+import { Font } from 'expo';
+import { BLACK,PRIMARY } from '../../styles/colors';
```

Comment:
code doesn't seem to be formatted correctly. Please run `yarn format` before committing.